packages/context/src/context.ts

```diff
@@ -340,6 +350,105 @@ export class Context extends EventEmitter {
     return this;
   }
 
+  /**
+   * Create a corresponding binding for configuration of the target bound by
+   * the given key in the context.
+   *
+   * For example, `ctx.configure('controllers.MyController').to({x: 1})` will
+   * create binding `controllers.MyController:$config` with value `{x: 1}`.
+   *
+   * @param key The key for the binding to be configured
+   */
+  configure<ConfigValueType = BoundValue>(
+    key: BindingAddress<unknown> = '',
```

Comment:
IIRC, `BindingAddress` has the template argument set to `unknown` by default, thus we should be able to simplify this line as follows:

```suggestion
    key: BindingAddress = '',
```

Same comment applies to other places below that are using `BindingAddress<unknown>` too.